netconf/transport.go

```diff
@@ -74,7 +78,12 @@ func (t *transportBasicIO) Send(data []byte) error {
 	}
 	dataInfo = append(dataInfo, data...)
 	dataInfo = append(dataInfo, seperator...)
-	t.Write(dataInfo)
+	writeLen, err := t.Write(dataInfo)
+	if err != nil {
+		if err.Error() == "EOF" && writeLen == 0 {
```

Comment:
You should not string check this.  if err == io.EOF should be done.    Also the len check is also not needed here.

Reply:
Ok, I'll change